README.md

```diff
@@ -159,11 +159,11 @@ This gem provides methods for accessing information from the [Channels HTTP API]
 
 The following methods are provided by the gem.
 
-- `pusher.channel_info('channel_name')` returns information about that channel.
+- `pusher.channel_info('channel_name', {info:"user_count,subscription_count"})` returns a hash describing the state of the channel([docs](https://pusher.com/docs/channels/library_auth_reference/rest-api#get-channels-fetch-info-for-multiple-channels-)).
```

Comment:
what does the `{info:"user_count,subscription_count"}` parameter do?

Reply:
There is a link to docs which shares more information on that account